plugins/com.google.cloud.tools.eclipse.appengine.localserver.test/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerDelegateTest.java

```diff
@@ -113,12 +111,7 @@ public void testCheckProjectFacets_deletedModule() throws CoreException {
   @Test
   public void testCheckConflictingId_defaultServiceIds() throws CoreException {
     delegate = getDelegateWithServer();
-    delegate.serviceIdFunction = new Function<IModule, String>() {
-      @Override
-      public String apply(IModule module) {
-        return "default";
-      }
-    };
+    delegate.serviceIdFunction = Functionsmodule -> "default";
```

Comment:
`Functionsmoudle` --> `module`?

Reply:
Oops.  That was from an experiment to see if there was a Guava-like `Functions.constant()`